src/gbd_mapping_generator/data.py

```diff
@@ -317,7 +312,7 @@ def get_coverage_gap_data():
     out = []
 
     for c in gbd.get_coverage_gap_list():
-        metadata = get_coverage_gap_metadata(c)
+        metadata = get_coverage_gap_metadata(c)['exposure']
```

Comment:
It needs a little more changes for affected causes/risks but I already have the change on my branch so let's leave it  for now. I need to make a PR for `gbd_mapping` soon anyways.